src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -326,10 +350,7 @@ def get_top_5_issues_by_count_for_file(
                                         sentry_filenames,
                                     ),
                                     Condition(
-                                        Function(
-                                            "arrayElement",
-                                            (Column("exception_frames.function"), i),
-                                        ),
+                                        stackframe_function_name(i),
```

Comment:
I'm don't think you need this `WHERE` condition since we have `multiIf` in the select. Maybe try removing this and evaluating a few test cases